scene/resources/material.cpp

```diff
@@ -1819,6 +1822,11 @@ void fragment() {)";
 	vec3 detail = mix(ALBEDO.rgb, ALBEDO.rgb * detail_tex.rgb, detail_tex.a);
 )";
 			} break;
+			case BLEND_MODE_PREMULT_ALPHA: {
+				// This is unlikely to ever be used for detail textures, and in order for it to function in the editor, another bit must be used in MaterialKey,
+				// but there are only 5 bits left, so I'm going to leave this disabled unless it's actually requested.
+				//code += "\tvec3 detail = (1.0-detail_tex.a)*ALBEDO.rgb+detail_tex.rgb;\n";
```

Comment:
@QbieShay I don't understand why this is commented out. Before the nuance is lost to the sands of time can you expand a little more? 5 bits left where?

https://github.com/godotengine/godot/blob/f91db3dc58f1d6a0cb63d591515183b5a45cd3ba/scene/resources/material.h#L326

The bits as far as I can tell are already spent allocating `BLEND_MODE_PREMULT_ALPHA`. Commenting it out just seems a little strange.

Reply:
Ah, that was probably taken from my original PR which didn't have the bits taken.  I was hesitant to waste a bit for this, but if the bit is already taken, this should be uncommented.